scarlet/__init__.py

```diff
@@ -8,5 +8,6 @@
 from .frame import *
 from .observation import *
 from .blend import *
+from .wavelet import *
```

Comment:
Please remove this line. We're not importing `fft` into the package namespace either

Reply:
done